srcpkgs/rebar3/template

```diff
@@ -10,12 +10,15 @@ maintainer="Noel Cower <ncower@gmail.com>"
 license="Apache-2.0"
 homepage="https://www.rebar3.org/"
 distfiles="https://github.com/erlang/rebar3/archive/${version}.tar.gz"
-checksum=7f358170025b54301bce9a10ec7ad07d4e88a80eaa7b977b73b32b45ea0b626e
+checksum=46d9991c8af1dc85a1de8edc6618c3e6dc9c3de60c67e8139a98d2e4c2d047ad
 
 do_build() {
 	./bootstrap
 }
 
 do_install() {
 	vbin rebar3
+	vlicense LICENSE
```

Comment:
no need to provide LICENSE for when it is Apache-2.0

Reply:
Nice. Removed that.